templates/ocsconsoleplugin.go

```diff
@@ -0,0 +1,30 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package templates
+
+import (
+	consolev1alpha1 "github.com/openshift/api/console/v1alpha1"
+)
+
+const (
+	ocpConsoleProxyAlias = "consoleproxy"
```

Comment:
```suggestion
	ocpConsoleProxyAlias = "odf-managed-service-prom"
```
Please use this. This is something we discussed before and is used in the UI as well.

Reply:
Done